DEVELOPING.md

```diff
@@ -18,10 +18,9 @@ This extension is built using the [gradle-intellij-plugin](https://github.com/Je
 
 1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git`
 2. Clone the lsp4ij repository: `git clone git@github.com:MicroShed/lsp4ij.git`
-3. Build lsp4ij: `./gradlew jar`
-4. Save it in your local Maven repository: `./gradlew publishToMavenLocal` (this also runs the `jar` task)
-2. Import this repository as a Gradle project in IntelliJ IDEA
-3. Run `./gradlew runIde --stacktrace`. A new IntelliJ IDEA window will launch with the Liberty Tools plugin installed to it. You can connect the IntelliJ IDEA debugger to this process to debug the plugin.
+3. Build lsp4ij and save it in your local Maven repository: `cd` to your local lsp4ij repository and run `./gradlew publishToMavenLocal`. Alternatively, build only the jar: `./gradlew jar`
```

Comment:
```suggestion
3. To build lsp4ij and save it in your local Maven repository, fun the following command from your local lsp4ij repository: `./gradlew publishToMavenLocal`. Alternatively, to build only the jar, run: `./gradlew jar`
```

Reply:
The person needs to build lsp4ij for this to work. "To build" sounds like it is optional. Are you assuming the person might already have done this? Indeed they only need to do it once. By the same token, "Alternatively" is not really an alternate because the build will not work if they do that unless they already know how to get the jar into their local maven repository. I just want to educate them how to build just the jar in case the need to one day. Since the step is not necessary should we remove the last sentence?

Reply:
Can we do something like this?

3. Build lsp4ij and save the jar to your local Maven repository by running the following command from your local clone of the lsp4ij repository: `./gradlew publishToMavenLocal`. If you only want to build the jar without publishing, you can run the following command: `./gradlew jar`.

Reply:
I'd say remove it, or add it as an optional sub-step with a little more context, so it's clear when/if you need to do that. I'll revise my suggestion

Reply:
@TrevCraw's  suggestion LGTM

Reply:
Decision to go with:

3. Build lsp4ij and save the jar to your local Maven repository by running the following command from your local clone of the lsp4ij repository: `./gradlew publishToMavenLocal`.

Reply:
Going with the idea below.